### PR TITLE
Permit partition size to be monitored via raw bytes instead of disk percentage

### PR DESCRIPTION
**Pull request by cadeef**
Permit partition size to be monitored via raw bytes instead of disk used percentages which don't work very well with modern disk sizes.

Enable with:
  linux:
    monitoring:
      disk:
        use_raw_size: True
        warn: 21474836480 # 20 GB
        crit: 5368709120  # 5 GB

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: linux/map.jinja [hunk 1]
```diff
@@ -206,6 +206,7 @@
               'crit': '20%',
         },
         'disk': {
+              'use_raw_size': False,
               'warn': '15%',
```
Comment (simonpasquier) [inline linux/map.jinja]: this is not directly related to the patch but you should remove '%'

**Comment by epcim** [comment]
Please rebase + explain while you ommit "%" char in setup.
